### PR TITLE
Tests for CAR endpoints.

### PR DESCRIPTION
**Pull request by atramors**
### Story:
[Create tests for car endpoints.](https://trello.com/c/DnPvfLTU/11-1-unit-tests-for-users-and-cars)

### Description:
We add more endpoints so it's time to cover them with tests.
Here what I did for now:
- Accordingly to the naming convention constants were named in _upper snail case_:
```diff
- test_user = {...}
+ TEST_USER = {...}
+ TEST_CAR = {...}
```
- Tests for car endpoints:
- [x] **GET**-,
- [x] **GET**- with filter, 
- [x] **DELETE**
- [x] **POST**-,
- [x] **PUT**- 
~~I'm not sure why **POST** and **PUT** don't work cause it's done in the same manner as **POST**- and **PUT**- for user:~~
Screenshot 2020-11-23 at 11 53 30

### CODE REVIEW

File: test.py [hunk 1]
```diff
@@ -1,99 +1,266 @@
...
+            "price": "1000000$",
+            "warranty": "12 months",
+            "fuel_consumption": "2l/100km",
+            "drivetrain": "4matic",
+            "date_added": "2020-10-21 15:02:54.770364",
+            "user_id": 999,
+        }
+        with app.test_client() as client:
+            MockCar.query.filter_by.update.return_value = CAR
+            response = client.put("/car/100000", json=CAR)
```
Comment (I159) [inline test.py:242]: Put a breakpoint here. What is the reason for the 422 response code? This is definitely about your payload.

Comment (I159) [reply test.py:242]: It is not a bad solution to pass id in the path parameters. Do you think your payload is consistent with the expectations of the endpoint?

**Review by I159 — CHANGES_REQUESTED** [review]
The situation is about your debugging skills. Go curious about your code, stick your nose wherever it is possible. The main difference between engineers' tech levels is the bravery to find new information about a system.

File: test.py (hunk 1 above)
Comment (atramors) [reply test.py]: Ok, I have found a problem in **test.py** and **scheme.py** already works good :)

**Review by I159 — APPROVED** [review]

**Pull request merged**